crates/libcontainer/src/rootfs/mount.rs

```diff
@@ -497,7 +497,7 @@ impl Mount {
                 err
             })?;
 
-            if src.is_file() {
+            if src.is_file() && !src.exists() {
```

Comment:
I don't think this is correct. This is the source of the bind mount, not the destination. The source will have to exist for bind mount to happen. The destination on the other hand can exist or not exist.

Reply:
🤦You are right. Thanks for your great reviewing. I have updated it.